Makefile

```diff
@@ -112,7 +112,11 @@ cctest: all
 
 v8:
 	tools/make-v8.sh
-	$(MAKE) -C deps/v8 $(V8_ARCH).$(BUILDTYPE_LOWER) $(V8_BUILD_OPTIONS)
+	ifeq ($(clang), off)
```

Comment:
Change this to the following, then you won't need the separate `clang` variable and it'll work properly:

```make
v8: export GYPFLAGS += -Dclang=0
v8:
	tools/make-v8.sh
	...
```